pkg/api/pullrequestpayloadqualification/v1/types.go

```diff
@@ -0,0 +1,93 @@
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/openshift/ci-tools/pkg/api"
+)
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// PullRequestPayloadQualificationRun represents the intent to run a battery of OCP release
+// payload validating jobs
+type PullRequestPayloadQualificationRun struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata"`
+
+	Spec   PullRequestPayloadTestSpec   `json:"spec"`
+	Status PullRequestPayloadTestStatus `json:"status"`
+}
+
+// PullRequestPayloadTestSpec specifies for which PR the payload qualification run was requested
+// and the list of individual jobs that should be executed
+type PullRequestPayloadTestSpec struct {
+	PullRequest PullRequestUnderTest        `json:"pullRequest"`
+	Jobs        []PullRequestPayloadJobSpec `json:"jobs"`
+}
+
+// PullRequestUnderTest describes the state of the repo that will be under test
+// This is a combination of the PR revision and base ref revision. Tested code
+// is the specific revision of the PR merged into the base branch with
+// a specific branch as a HEAD
+type PullRequestUnderTest struct {
+	// Org is something like kubernetes or k8s.io
```

Comment:
can you explain this in terms of github.com/foo/bar or is it related to the golang package?